SPECS/cmake/cmake.spec

```diff
@@ -1,8 +1,8 @@
 %global major_version 3
 Summary:        Cmake
 Name:           cmake
-Version:        3.21.4
-Release:        10%{?dist}
+Version:        3.27.4
```

Comment:
Looks like 3.28.1 has already been released. Should we try upgrading directly to that?